### PR TITLE
Plant Epic Issue Template

### PR DESCRIPTION
**Pull request by Tricera-Manuel-Kersic**
Introduce plant epic issue template for an overview (!) of all tasks coming up for a battery plant and to track its progress

### CODE REVIEW

**Comment by Tricera-Manuel-Kersic** [comment]
This is the list that came up in my mind and my experience of planning and commissioning our battery plants. You all have different views on theses processes and I would like to have a good (and possibly generic) overview template that includes most of the tasks for all of us. Would be great if you could try to find major gaps (not too detailed)) in this list related to your tasks so far.

File: .github/ISSUE_TEMPLATE/plant-epic.md [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+---
+
+<!--
+This is a proposed minimum (!) list of plant related tasks. Feel free to add more tasks relevant for the specific plant or to customize. But focus on an overview in this epic, i.e. do not go too much into details.
+Each bullet point can easily be changed to an issue and described in more detail there.
+-->
+
+### Detailed Engineering
+- [ ] Select hardware
+- [ ] Create network plan
```
Comment (Tricera-Hendrik-Weiss) [inline .github/ISSUE_TEMPLATE/plant-epic.md]:
```suggestion
- [ ] Create network plan
- [ ] Select hardware
```
We are running into trouble (redundancy tasks, steag/SENS fiber optics, ...)  a lot as we don't have a network plan when ordering hardware.

File: .github/ISSUE_TEMPLATE/plant-epic.md [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+- [ ] Agree on data provision to the customer (FTP, ... )
+
+### Specific new software implementation requirements (link issues and/or prs!)
+- [ ] ...
+
+### Prepare commissioning
+- [ ] Create IP list including port forwardings to other network segments (e.g. smartblue ppc)
+- [ ] Prepare software commissioning protocol
+- [ ] Setup wireguard VPN configuration
+- [ ] Configure router
```
Comment (Tricera-Hendrik-Weiss) [inline .github/ISSUE_TEMPLATE/plant-epic.md]:
```suggestion
- [ ] Configure router
  - [ ] Setup login credentials
  - [ ] Setup IP address
  - [ ] Setup DHCP Server
  - [ ] Setup wireguard VPN configuration
```

File: .github/ISSUE_TEMPLATE/plant-epic.md [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+- [ ] Create project configurations
+  - [ ] bcc
+  - [ ] ems
+  - [ ] (ucm)
+- [ ] Configure IO systems (RevPi, Moxa)
+- [ ] Configure server
+- [ ] Setup kubernetes cluster
+- [ ] Check datalogging in influx
+- [ ] Check grafana
```
Comment (Tricera-Hendrik-Weiss) [inline .github/ISSUE_TEMPLATE/plant-epic.md]:
```suggestion
- [ ] Configure server
  - [ ] Setup IP address of OS and IPMI
  - [ ] Setup kubernetes cluster
  - [ ] Rollout: MetalLB, Traefik, Prometheus, Loki, MariaDB, EMQX, Grafana, InfluxDB, Telegraf
  - [ ] Rollout: ESF
  - [ ] Rollout: BCC
  - [ ] Rollout: EMS
  - [ ] Check datalogging in influx
  - [ ] Check grafana access
  - [ ] Optional: Setup Customer specific alerts
```

**Review by Tricera-Hendrik-Weiss — CHANGES_REQUESTED** [review]

File: .github/ISSUE_TEMPLATE/plant-epic.md [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+
+---
+
+<!--
+This is a proposed minimum (!) list of plant related tasks. Feel free to add more tasks relevant for the specific plant or to customize. But focus on an overview in this epic, i.e. do not go too much into details.
+Each bullet point can easily be changed to an issue and described in more detail there.
+-->
+
+### Detailed Engineering
+- [ ] Select hardware
```
Comment (Tricera-Paul-Klotsche) [inline .github/ISSUE_TEMPLATE/plant-epic.md:19]:
```suggestion
- [ ] Collect SW related requirements from offer, contract and other binding documents
- [ ] Select hardware
```

File: .github/ISSUE_TEMPLATE/plant-epic.md [hunk 5]
```diff
@@ -0,0 +1,60 @@
...
+- [ ] Create network plan
+- [ ] Agree on network operator interface (to ppc)
+- [ ] Create operational control concept
+- [ ] Create power plant controller (*EZA-Regler*) concept (if necessary)
+- [ ] Create *Regelungskonzept* for plant certificate, (if necessary)
+- [ ] Agree on marketer interface (if necessary)
+- [ ] Agree on customer control interface (if necessary)
```
Comment (Tricera-Paul-Klotsche) [inline .github/ISSUE_TEMPLATE/plant-epic.md:25]:
```suggestion
- [ ] Agree on customer control interface (if necessary)
- [ ] Agree on interface to site periphery, i.e. fire detection, building management system (DE: GLT) (if necessary)
```

File: .github/ISSUE_TEMPLATE/plant-epic.md [hunk 6]
```diff
@@ -0,0 +1,60 @@
...
+- [ ] Agree on data provision to the customer (FTP, ... )
+
+### Specific new software implementation requirements (link issues and/or prs!)
+- [ ] ...
```
Comment (Tricera-Paul-Klotsche) [inline .github/ISSUE_TEMPLATE/plant-epic.md:34]:
```suggestion
- [ ] Order PPC with project specific requirements
- [ ] ...
```

File: .github/ISSUE_TEMPLATE/plant-epic.md [hunk 7]
```diff
@@ -0,0 +1,60 @@
...
+- [ ] Configure server
+- [ ] Setup kubernetes cluster
+- [ ] Check datalogging in influx
+- [ ] Check grafana
+
+### Commissioning (goes hand in hand with the commissioning protocol)
+- [ ] Establish wireguard VPN connection to the plant
+- [ ] Check communication to all devices in the IP list
+- [ ] Check connections to the IO systems (RevPis, Moxas)
+- [ ] Check communication with smartblue router and the ppc
```
Comment (Tricera-Paul-Klotsche) [inline .github/ISSUE_TEMPLATE/plant-epic.md]:
```suggestion
- [ ] Check communication with smartblue router and the ppc
- [ ] Configure Inverters according to E.9
- [ ] Check mapping of SGs and inverters in BCC. 
- [ ] Test attaching all SGs and inverter
- [ ] Perform power test (attach and apply some power via bcc)
```

File: .github/ISSUE_TEMPLATE/plant-epic.md [hunk 8]
```diff
@@ -0,0 +1,60 @@
...
+- [ ] Test interfaces with
+  - [ ] marketer (if interfaced to ems)
+  - [ ] network operator (mostly smartblue's part)
+  - [ ] (customer)
+- [ ] Perform power test (attach and apply some power via bcc)
```
Comment (Tricera-Paul-Klotsche) [inline .github/ISSUE_TEMPLATE/plant-epic.md]:
```suggestion
```

File: .github/ISSUE_TEMPLATE/plant-epic.md [hunk 9]
```diff
@@ -0,0 +1,60 @@
...
+- [ ] Check grafana
+
+### Commissioning (goes hand in hand with the commissioning protocol)
+- [ ] Establish wireguard VPN connection to the plant
+- [ ] Check communication to all devices in the IP list
+- [ ] Check connections to the IO systems (RevPis, Moxas)
+- [ ] Check communication with smartblue router and the ppc
+- [ ] Test interfaces with
+  - [ ] marketer (if interfaced to ems)
+  - [ ] network operator (mostly smartblue's part)
```
Comment (Tricera-Paul-Klotsche) [inline .github/ISSUE_TEMPLATE/plant-epic.md]:
```suggestion
  - [ ] network operator ("Bittest") (mostly smartblue's part)
```

File: .github/ISSUE_TEMPLATE/plant-epic.md [hunk 10]
```diff
@@ -0,0 +1,60 @@
...
+  - [ ] (customer)
+- [ ] Perform power test (attach and apply some power via bcc)
+- [ ] Test whole control chain, possibly with smartblue (e.g. marketer/network operator -> ems -> ppc -> bcc)
```
Comment (Tricera-Paul-Klotsche) [inline .github/ISSUE_TEMPLATE/plant-epic.md]:
```suggestion
- [ ] Configure PPC according to E.9
- [ ] Test whole control chain, possibly with smartblue (e.g. marketer/network operator -> ems -> ppc -> bcc)
- [ ] Functional test with network operator
```

File: .github/ISSUE_TEMPLATE/plant-epic.md [hunk 11]
```diff
@@ -0,0 +1,60 @@
...
+- [ ] Establish data provision to customer (e.g. FTP push to customer's endpoint or smartblue portal)
+
+### Performance Tests
+- [ ] Run capacity test
+- [ ] Run double hump test (*Doppelhöckertest*) (if necessary)
```
Comment (Tricera-Paul-Klotsche) [inline .github/ISSUE_TEMPLATE/plant-epic.md]:
```suggestion
- [ ] FCR: Run double hump test (*Doppelhöckertest*) and 4h test (if necessary)
- [ ] Compliance test ("Konformitätstest") 
- [ ] Handover to O&M / customer, documentation
```

**Review by Tricera-Paul-Klotsche — CHANGES_REQUESTED** [review]

**Pull request merged**